Multisig/UI/ClaimToken/NavigatingDAOViewController.swift

```diff
@@ -58,4 +84,42 @@ class NavigatingDAOViewController: UIViewController {
         completion?()
     }
 
+    func openUrlOnTap(link: link, label: UILabel) {
+        var tapRecognizer: UITapGestureRecognizer
+        switch link {
+        case .discuss: tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(discussTap(sender:)))
+        case .propose: tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(proposeTap(sender:)))
+        case .chat:  tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(chatTap(sender:)))
+        }
+        label.addGestureRecognizer(tapRecognizer)
+    }
+
+    @objc
+    func discussTap(sender: UITapGestureRecognizer) {
+        guard let url = URL(string: "https://forum.gnosis-safe.io/") else {
+            return
+        }
+        openInSafari(url)
+    }
+
+    @objc
+    func proposeTap(sender: UITapGestureRecognizer) {
+        guard let url = URL(string: "https://docs.google.com/document/d/1t79exFjYZQO80yDc8X2S61d4Sc1_QjBwnzZgTSWqoUo/edit#heading=h.ao7vqnfg6hyh") else {
+            return
+        }
+        openInSafari(url)
+    }
+
+    @objc
+    func chatTap(sender: UITapGestureRecognizer) {
+        guard let url = URL(string: "https://discord.com/invite/AjG7AQD9Qn") else {
```

Comment:
also for discord link we could define config value and have additional struct in `AppConfiguration`

```
struct Claim {
```